demos/_unit-test/grid-rowclick.php

```diff
@@ -17,7 +17,7 @@
 require_once __DIR__ . '/../init-app.php';
```

Comment:
please keeps conversations in threads

> I checked demos for a test case.
> https://ui.agiletoolkit.org/demos/collection/grid.php
> https://ui.agiletoolkit.org/demos/collection/crud.php
> Unfortunately, none of them seems to have a working delete action.
> `grid.php` has the delete button commented, if restored, the row does not fade away and the button also appears below the grid.
> `crud.php` deletion just triggers a read-only error without hiding the row.

This is expected, as least `crud.php`, web demos are read only as they are public thanks to https://github.com/atk4/ui/blob/5.0.0/demos/init-db.php#L25